ui/app/services/keyboard.js

```diff
@@ -110,6 +111,7 @@ export default class KeyboardService extends Service {
       {
         label: 'Go to Variables',
         action: () => this.router.transitionTo('variables'),
+        rebindable: true,
```

Comment:
Side-effect: the "go to variables" key command wasn't re-bindable! I must've forgotten during Vars work.